.github/workflows/python-tests.yml

```diff
@@ -24,12 +24,22 @@ jobs:
           source /home/olender/Firedrakes/newest3/firedrake/bin/activate
```

Comment:
For local use you can copy it (https://stackoverflow.com/questions/34993200/copy-complete-virtualenv-to-another-pc). 

For a docker build I think it would be harder, unless you actually copy it to the container. Building a container while calling an older version of Firedrake might also work, but you probably would have to alter the install script, which would be complex.